addons/event_sale/models/event_ticket.py

```diff
@@ -1,108 +1,10 @@
-                ticket.price = 0
-
-    @api.depends('product_id')
-    def _compute_description(self):
-        for ticket in self:
-            if ticket.product_id and ticket.product_id.description_sale:
-                ticket.description = ticket.product_id.description_sale
-            # initialize, i.e for embedded tree views
-            if not ticket.description:
-                ticket.description = False
-
-    # TODO clean this feature in master
-    # Feature broken by design, depending on the hacky `_get_contextual_price` field on products
-    # context_dependent, core part of the pricelist mess
-    # This field usage should be restricted to the UX, and any use in effective
-    # price computation should be replaced by clear calls to the pricelist API
-    @api.depends_context(*PRICE_CONTEXT_KEYS)
-    @api.depends('product_id', 'price')
-    def _compute_price_reduce(self):
-        for ticket in self:
-            contextual_discount = ticket.product_id._get_contextual_discount()
-            ticket.price_reduce = (1.0 - contextual_discount) * ticket.price
-
-    def _init_column(self, column_name):
-        if column_name != "product_id":
-            return super(EventTemplateTicket, self)._init_column(column_name)
-
-        # fetch void columns
-        self.env.cr.execute("SELECT id FROM %s WHERE product_id IS NULL" % self._table)
-        ticket_type_ids = self.env.cr.fetchall()
-        if not ticket_type_ids:
-            return
-
-        # update existing columns
-        _logger.debug("Table '%s': setting default value of new column %s to unique values for each row",
-                      self._table, column_name)
-        default_event_product = self.env.ref('event_sale.product_product_event', raise_if_not_found=False)
-        if default_event_product:
-            product_id = default_event_product.id
-        else:
-            product_id = self.env['product.product'].create({
-                'name': 'Generic Registration Product',
-                'list_price': 0,
-                'standard_price': 0,
-                'type': 'service',
-            }).id
-            self.env['ir.model.data'].create({
-                'name': 'product_product_event',
-                'module': 'event_sale',
-                'model': 'product.product',
-                'res_id': product_id,
-            })
-        self.env.cr._obj.execute(
-            f'UPDATE {self._table} SET product_id = %s WHERE id IN %s;',
-            (product_id, tuple(ticket_type_ids))
-        )
-
-    @api.model
-    def _get_event_ticket_fields_whitelist(self):
-        """ Add sale specific fields to copy from template to ticket """
-        return super(EventTemplateTicket, self)._get_event_ticket_fields_whitelist() + ['product_id', 'price']
 
 
 class EventTicket(models.Model):
     _inherit = 'event.event.ticket'
     _order = "event_id, sequence, price, name, id"
 
-    # product
     price_reduce_taxinc = fields.Float(
```

Comment:
If you move price fields on type ticket, why not on event ticket ? Semes logical to move them with the product, isn't it ?